content/news/2018-07-30-navpi-memory-management-v1-0-9-release-notes.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: news
+generated_by_cms: true
+title: NavPi Memory Management & v1.0.9 Release Notes
+author: zanuka
+date: '2018-07-30T10:39:01+12:00'
+feature_image: /images/uploads/navpi3.jpg
+categories:
+  - Nav News
+  - NavPi
+---
+## NavPi Memory Management
+
+As most veteran NavPi users know, there has been a significant issue with the memory resources being completely maxed out while running the NavCoin daemon. This has resulted in the NavPi’s default Chromium WebUI being nearly unusable and has also caused the daemon to crash on new installations and upgrades.  Advanced users of the NavPi have tweaked their systems for optimised performance, but for new users the initial setup and configuration mods are most certainly daunting tasks. It’s been a frustrating issue for everyone and the core team has been focused on solving the problem by engineering both short and long term solutions.
+
+### The problem; memory is maxed
+
+When the NavCoin daemon starts, it loads the entire chainstate database in memory. This causes a large spike in memory usage until that operation is complete. Existing users have reported that after updating to the latest version of NavCoin, the memory spike can cause the Chromium WebUI to be nearly unusable in addition to the navcoind daemon simply crashing. 
```

Comment:
Some users might not know what the chainstate is so adding something like this could help clarify: "the entire chainstate database (a database containing information on the entire blockchain)".
Also this text is basically a repeat of what you have in the first paragraph not sure if you caught it: "the memory spike can cause the Chromium WebUI to be nearly unusable in addition to the navcoind daemon simply crashing. "